boards/common/stm32-stlink/Makefile.include

```diff
@@ -0,0 +1,14 @@
+# configure the serial terminal
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# Default is to use ST-link v2-1 adapter (used by all Nucleos)
+export DEBUG_ADAPTER ?= stlink
+export STLINK_VERSION ?= 2-1
```

Comment:
How about to more this line to the top of `makefiles/tools/openocd-adapters/stlink.inc.mk` and add doc that version 2-1 is the default?